LICENSE-MIT

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Jon Magnuson
```

Comment:
I knew you were a time traveler.

Reply:
2017 was a good year for making files you would later blindly copy over into new projects.